### PR TITLE
Add PDF Stitcher application

### PR DESCRIPTION
**Pull request by nbenitez**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/cfcurtis/pdfstitcher/issues/27#issuecomment-1032000524
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* Upstream is only providing binaries for Windows and Mac

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.github.cfcurtis.pdfstitcher

**Comment by hfiguiere** [comment]
appdata fail to validate 
```
builddir/files/share/appdata/com.github.cfcurtis.pdfstitcher.appdata.xml: FAILED:
• style-invalid         : Content before <ul> is too short [8], at least 20 characters required
Validation of files failed
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.github.cfcurtis.pdfstitcher.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        {
+            "name": "python3-wxPython",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"wxPython==4.1.1\" --no-build-isolation"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/b0/43/3e286c93b9fa20e233d53532cc419b5aad8a468d91065dbef4c846058834/Pillow-9.0.0.tar.gz",
```
[the remark this answers is not in the document]
Comment (nbenitez) [reply com.github.cfcurtis.pdfstitcher.json]: Removed.

**Comment by nbenitez** [comment]
bot, build com.github.cfcurtis.pdfstitcher

**Pull request closed without merging**